java/c/src/main/java/org/apache/arrow/c/Data.java

```diff
@@ -314,4 +325,14 @@ public static VectorSchemaRoot importVectorSchemaRoot(BufferAllocator allocator,
     }
     return vsr;
   }
+
+  /**
+   * Import an ArrowArrayStream as an {@link ArrowReader}.
+   * @param allocator Buffer allocator for allocating the output data.
+   * @param stream C stream interface struct to import.
+   * @return Imported reader
+   */
+  public static ArrowReader importStream(BufferAllocator allocator, ArrowArrayStream stream) {
```

Comment:
Is there a reason for the naming discrepancy (`importStream` vs. `exportArrayStream`)?